src/components/molecules/event/eventBody/EventBody.tsx

```diff
@@ -289,7 +297,15 @@ export const EventInfo: React.FC<{ event: Event; role: RoleOptions }> = ({
   }, [event.eid, event?.maxParticipants]);
 
   useEffect(() => {
-    getNumberOfParticipants();
+    // only fetch participants list when the list is actually used
+    // the list/number of participants should only be displayed for admins and events with no cap
+    if (role === Roles.admin || event.maxParticipants == null) {
+      getNumberOfParticipants();
+      return;
+    }
+    console.log(event.maxParticipants);
```

Comment:
could be removed

Reply:
![](https://media.giphy.com/media/26hishnZ9EcHI6eiY/giphy.gif)